spec/queries/display_ads/filtered_ads_query_spec.rb

```diff
@@ -129,7 +129,7 @@ def filter_ads(**options)
     let!(:external_ad) { create_display_ad organization_id: organization.id, type_of: :external }
     let!(:other_external) { create_display_ad organization_id: other_org.id, type_of: :external }
 
-    it "always shows :community ad if matching, otherwise shows in_house/external" do
+    it "always shows :community ad if matching, otherwise shows in_house/external", :aggregate_failure do
```

Comment:
This one I want some more info on why it's flaky. I'm hoping `aggregate_failure will tell me